app/models/mailing_list/steps/postcode.rb

```diff
@@ -14,6 +14,11 @@ class Postcode < ::GITWizard::Step
       def optional?
         true
       end
+
+      def skipped?
+        # Don't show the postcode if you are not a UK Citizen AND your location is not in the UK.
+        !other_step(:citizenship).uk_citizen? && !other_step(:location).inside_the_uk?
```

Comment:
this _might_ be more readable, i don't know

```
[
  other_step(:citizenship).uk_citizen?,
  other_step(:location).inside_the_uk?
].none?
```

Reply:
horses for courses 😃